hummingbot/strategy/pure_market_making/pure_market_making.pyx

```diff
@@ -837,6 +852,8 @@ cdef class PureMarketMakingStrategy(StrategyBase):
         cdef:
             str order_id = order_completed_event.order_id
             limit_order_record = self._sb_order_tracker.c_get_limit_order(self._market_info, order_id)
+        if self._price_is_mid:
+            self._last_price = float(order_completed_event.price)
```

Comment:
This and the one below is the section where I would need some help. `limit_order_record` is `None` if the order is not one the PMM has placed, but `order_completed_event` has no `price` attribute. How would I then go about keeping track of the last price? Maybe there is a much better way I am not seeing?

Reply:
The `last_trade_price` is actually the last trade price of the market, not user's order price.
https://github.com/CoinAlpha/hummingbot/issues/2036
We will need to get the above issue done first before you can continue on this, I'll raise this as a priority issue now.

Reply:
I see. That would simplify the task. Thank you!